src/main/java/edu/wpi/grip/ui/pipeline/PipelineView.java

```diff
@@ -152,10 +152,12 @@ private OutputSocketView findOutputSocketView(OutputSocket socket) {
     }
 
     /**
+     * This function is public because it is used by {@link edu.wpi.grip.ui.preview.PreviewsView#onSocketPreviewChanged}
+     * to order the list of previews.
+     *
```

Comment:
I don't think you need to document everywhere a function is used, since it's possible that this comment will be out of date if we change where it's called. 

If we want to find a method's usages, IntelliJ has features for that.